kong/plugins/http-log/handler.lua

```diff
@@ -153,6 +155,14 @@ function HttpLogHandler:log(conf)
 
   local queue_id = get_queue_id(conf)
   local q = queues[queue_id]
+  if q and not tx_deepcompare(conf, q.conf) then
```

Comment:
I think `http-log` should not have such problem as there is no nested config fields inside this plugin and every time a new config will result in new `queue_id`?